releases.json

```
@@ -114,4 +114,10 @@
     "extended_eol": false,
     "security_eol": "2028-02-28"
   }
```

```suggestion
  },
```
Missing a comma, resulting in invalid JSON